tdrs-backend/tdpservice/conftest.py

```diff
@@ -91,6 +91,12 @@ def stt():
     return STTFactory.create()
 
 
+@pytest.fixture
+def other_stt():
```

Comment:
This was needed to test downloading reports for an STT that a data prepper does not belong to.